Contents/Perks/ModPerk.cs

```diff
@@ -183,17 +193,21 @@ public class UnstableImbue : Perk {
 	public class AlchemistKnowledge : Perk {
 		public override void SetDefaults() {
 			CanBeStack = true;
-			Tooltip = "+ Mysterious potion are slightly better than before";
+			Tooltip = "" +
+				"+ Mysterious potion are slightly better than before" +
+				"\n+ Increases amount of potion drop from lootbox by 1";
 			StackLimit = 3;
 		}
 		public override void ResetEffect(Player player) {
 			player.GetModPlayer<MysteriousPotionPlayer>().PotionPointAddition += 1 * StackAmount;
+			player.GetModPlayer<ChestLootDropPlayer>().PotionTypeAmountAddition += 1 * StackAmount;
```

Comment:
`1 * StackAmount` is same as just `StackAmount`

Reply:
oh yeah, that true, will change it